programs/ido-program/tests/ido-program.ts

```diff
@@ -0,0 +1,466 @@
+import * as anchor from "@project-serum/anchor";
```

Comment:
Update to the new package
```suggestion
import * as anchor from "@coral-xyz/anchor";
```